src/Serilog.Settings.AppSettings/Settings/AppSettings/AppSettingsSettings.cs

```diff
@@ -56,7 +56,7 @@ public void Configure(LoggerConfiguration loggerConfiguration)
             else
             {
                 settings = ConfigurationManager.AppSettings.AllKeys
-                    .Select(k => new KeyValuePair<string, string>(k, ConfigurationManager.AppSettings[k]));
+                    .Select(k => new KeyValuePair<string, string>(k, ConfigurationManager.AppSettings[k]!));
```

Comment:
The ```!``` here is going on the assumption that there will always be a value for a specified key, when the key in question is from ```AllKeys```